cmd/swarm/main.go

```diff
@@ -248,6 +248,15 @@ Removes a path from the manifest
 Cleans database of corrupted entries.
 `,
 		},
+		{
+			Action:    importdb,
+			Name:      "importdb",
+			Usage:     "Imports chunks from one datastore to another",
+			ArgsUsage: "<source-datadir> s",
```

Comment:
This `ArgsUsage` looks incomplete, should it be `<source-datadir> <target-datadir>`?